modulefiles/ufs_hera.intel.lua

```diff
@@ -2,7 +2,7 @@ help([[
 loads UFS Model prerequisites for Hera/Intel
 ]])
 
-prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.4.1/envs/ufs-pio-2.5.10/install/modulefiles/Core")
+prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.5.0/envs/unified-env-noavx512/install/modulefiles/Core")
 prepend_path("MODULEPATH", "/scratch1/NCEPDEV/jcsda/jedipara/spack-stack/modulefiles")
```

Comment:
I think this MODULEPATH is not needed anymore

Reply:
@DusanJovic-NOAA I forgot to take this out before testing, is it ok to leave it so that we don't have to rerun the tests on Hera?